platform/MCU/NRF51/SPARK_Firmware_Driver/src/hw_config.c

```diff
@@ -146,6 +147,9 @@ void FLASH_Begin(uint32_t sFLASH_Address, uint32_t fileSize)
 {
     //OTA_FLASHED_Status_SysFlag = 0x0000;
     //Save_SystemFlags();
+#if PLATFORM_ID==103
```

Comment:
Nah. That would kill the system timer, 0. Let's see ... 'MAX_ TIMERS=11, being 0..10. We want to stop 1 through 10. i will stop at 10 ... oh. Oops. I'm going one too high. Should be ...

    for (int i=1; ...) 'stop(i)

Will push the fix when I get back to a computer. (on iPhone)